exchanges/stream/websocket.go

```diff
@@ -56,6 +56,8 @@ var (
 	errReadMessageErrorsNil                 = errors.New("read message errors is nil")
 	errWebsocketSubscriptionsGeneratorUnset = errors.New("websocket subscriptions generator function needs to be set")
 	errSubscriptionsExceedsLimit            = errors.New("subscriptions exceeds limit")
+	errSubscriptionsNotAdded                = errors.New("subscriptions not added")
+	errSubscriptionsNotRemoved              = errors.New("subscriptions not removed")
```

Comment:
Now that these are being returned by `Connect()`, I think it would be nice to export these so you can help distinguish an error from connecting or an error for subscribing and act accordingly

Reply:
[done](https://github.com/thrasher-corp/gocryptotrader/pull/1717/commits/aca87c35d233d709a78ffb60b8085ed8e5d7781d)